src/mesh/parallel/fci.cxx

```diff
@@ -73,11 +73,16 @@ FCIMap::FCIMap(Mesh& mesh, int offset_, BoundaryRegionPar* boundary, bool zperio
   auto k_corner = Tensor<int>(map_mesh.LocalNx, map_mesh.LocalNy, map_mesh.LocalNz);
 
   // Index-space coordinates of forward/backward points
-  Field3D xt_prime(&map_mesh), zt_prime(&map_mesh);
+  Field3D xt_prime(&map_mesh);
+  Field3D zt_prime{emptyFrom(xt_prime)};
```

Comment:
Do we need `emptyFrom` here if we're just going to use `Mesh::get`? Unless that checks that the fields are compatible?

Reply:
Ah, this use of `emptyFrom` was only useful when I was calling the constructor for `xt_prime` with more arguments, that then wanted to be copied. It doesn't add anything any more, so will revert.